File: BP_API/BP_API/DTO/ClienteDTO.cs
namespace BP_API.DTO
{
    public class ClienteDTO : PersonaDTO
    {
        public int IdCliente { get; set; }
        public string Contrasena { get; set; }
        public int Estado { get; set; } 
        public string EstadoDescripcion { get; set; }
    }
}

File: BP_Front/BP/src/app/Interfaces/ClienteDTO.ts
export interface ClienteDTO {
    idCliente: number;
    id: number;
    nombre: string;
    genero: number;
    generoDescripcion: string;
    fechaNacimiento: string; // Utilizamos string por formato ISO (YYYY-MM-DD)
    identificacion: string;
    tipoIdentificacion: number;
    tipoIdentificacionDescripcion: string;
    direccion: string;
    telefono: string;
    contrasena: string;
    estado: number;
    estadoDescripcion: string;
    edad: number;
  }